news/1527.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Fix multipackaging
+
+- Correct relative paths between shared dependencies
+- Fix issue when copying dependency files in the bootloader.
```

Comment:
Several remarks on this :-)

1. This rst-markup might not work as intented due to limitations og towncrier.

2. 1527 was the original issue, and 4303 is the pull-request. Thus there shall be several news fragments:
   - 1527.bugfix: about fixing multiprocessing (I suggest mentioning `MERGE` here)
   - 4303.bugfix: copy of that file
   - 4303.bootloader: about the bootloader issue (which issue? please elaborate)

3. Need a newsfragment for the 4th commit? Just name it e.g `fpnull.bootloader.rst`

Reply:
Done. I'm not 100% sure on the wording